x/evm/keeper/keeper.go

```diff
@@ -416,6 +437,22 @@ func (k *Keeper) SetReplayInitialHeight(ctx sdk.Context, h int64) {
 	k.setInt64State(ctx, types.ReplayInitialHeight, h)
 }
 
+func (k *Keeper) GetBlockTestHeight(ctx sdk.Context) int64 {
```

Comment:
can we add a new file under `keeper/` called something like `block_test.go` and put these methods there?